android/app/src/main/java/org/fossasia/openevent/viewmodels/SpeakersListFragmentViewModel.java

```diff
@@ -27,8 +27,10 @@ public SpeakersListFragmentViewModel() {
 
     public LiveData<List<Speaker>> getSpeakers(int sortType, String searchText) {
         setSearchText(searchText);
-        if (sortType != speakersListSortType) {
-            filterableRealmLiveData = RealmDataRepository.asFilterableLiveData(realmRepo.getSpeakers(sortOrderSpeaker()));
+        if (sortType != speakersListSortType || filterableRealmLiveData == null) {
+            if(filterableRealmLiveData == null) {
```

Comment:
Please remove this null check. I thought it needs to be created only if the filterableRealmLiveData is null but it is also using ```sortOrderSpeaker()``` inside the realmRepo.getSpeakers call.